virttest/qemu_vm.py

```diff
@@ -4579,8 +4596,8 @@ def save_to_file(self, path):
             self.MIGRATE_TIMEOUT, 2, 2,
             "Waiting for save to %s to complete" % path)
         # Restore the speed and downtime to default values
-        self.monitor.migrate_set_speed(str(32 << 20))
-        self.monitor.migrate_set_downtime(0.03)
+        qemu_migration.set_speed(self, str(32 << 20))
+        qemu_migration.monitor.set_downtime(self, 0.03)
```

Comment:
Hi @YongxueHong I believe here you meant to add the line `qemu_migration.set_downtime` instead of `qemu_migration.monitor`, should I send a fixing PR?

Reply:
Hi @pevogam 
Would you like to explain what is the purpose of your fixing PR?
Thanks.

Reply:
Hi @YongxueHong as I explained above, there exists no `qemu_migration.monitor` and you probably meant `qemu_migration.set_downtime` above. Would you agree with my suggestion?

Reply:
Hi @pevogam
oh~~, I misunderstand your point, my bad. 
Yes, Sure. Your suggestion is good for me.
And thank you for pointing out it.

Reply:
Hehe alright so would you want me to push a PR with the fix or you prefer having your own PR?

Reply:
> Hehe alright so would you want me to push a PR with the fix or you prefer having your own PR?

Yeah, I will send a PR to cover it by myself.
So sorry for the inconvenience.

Reply:
Hi @pevogam 
I have sent a PR for this issue. https://github.com/avocado-framework/avocado-vt/pull/3488
Would you like to help to review it?
Thanks a lot.

Reply:
Sure, sorry for taking some time to notice this.